src/rebar3_hex_publish.erl

```diff
@@ -347,6 +354,14 @@ include_files(Name, AppDir, AppDetails) ->
     AppSrcBinary = rebar_utils:to_binary(lists:flatten(io_lib:format("~tp.\n", [AppSrc]))),
     lists:keystore(AppFileSrc, 1, WithIncludes, {AppFileSrc, AppSrcBinary}).
 
+maybe_print__checkouts_warnings(AppDir) ->
+    {HasCheckouts, Checkouts} = has__checkouts_for(AppDir),
+    HasCheckouts andalso
+        rebar_log:log(warn, "~p directory found; this might interfere with publishing", [Checkouts]).
+
+has__checkouts_for(AppDir) ->
```

Comment:
Nit : same, remove the `_` before `checkouts`. I realize the dir is called `_checkouts`, but not sure it makes sense as part of a function name. 🤷

Reply:
Yeah, I wanted to be correct, but if it pleases you otherwise I'll change it.

Reply:
"Fixed", rebased and force-pushed.